client/core/core.go

```diff
@@ -2202,6 +2202,11 @@ func (c *Core) loadDBTrades(dc *dexConnection, crypter encrypt.Crypter, failed m
 	errs := newErrorSet(dc.acct.host + ": ")
 	ready := make([]*trackedTrade, 0, len(dc.trades))
 	for _, trade := range trades {
+		// The DB loads refunded and revoked matches, so filter those out.
+		if !trade.isActive() {
+			log.Tracef("Loaded inactive trade %v from the DB.", trade.ID())
```

Comment:
I started making `(*BoltDB).DEXOrdersWithActiveMatches` apply some of the same inactive checks (https://github.com/decred/dcrdex/pull/574/files#diff-bc5a163343b4e039fb94cf91fe176b46R626), but I thought I should leave anything beyond a status==MatchComplete check to the DB _consumer_, especially since we're still ironing out the refund and redeem paths and status changes.